src/intl/en/quizzes.json

```diff
@@ -0,0 +1,3 @@
+{
+  "quiz-test-your-knowledge": "Test your knowledge"
```

Comment:
Is this just a placeholder? I'm guessing this PR doesn't include internationalization support to start?

Reply:
Looks like this was teed up during the initial draft, presumedly as a placeholder to prep for future internationalization. Happy to just revert this for now since it's the only string set up this way.